topology/Penn State University/PSU LIGO/PSU-LIGO.yaml

```
@@ -33,7 +33,7 @@ Resources:
 
   PSU-LIGO-CACHE:
     Active: false
-    Description: This is a cache for public and LIGO data at Penn State
+    Description: This is a cache for public and LIGO data at Penn State, restrict to gwave cluster
```

How about a small wording tweak?

```suggestion
    Description: This is a cache for public and LIGO data at Penn State, restricted to the gwave cluster
```